scripts/lint_prowjobs/main.go

```diff
@@ -83,10 +83,11 @@ func EnvVarsCheck(jc *JobConstants) presubmitCheck {
 			for _, env := range container.Env {
 				if index, exists := jc.envVarExist(env.Name); exists {
 					// check deepequal in case we decide to support EnvVarSource values in the future
-					if jobs.IsCuratedPackagesPresubmit(presubmitConfig.JobBase.Name) {
-						jc.EnvVars[index].Value = "s3://codebuildprojectstack-be-pipelineoutputartifactsb-jvwhrzx05xwq"
+					artifacts_bucket_env := jc.EnvVars[index]
```

Comment:
nit:
```suggestion
					jcEnvVar := jc.EnvVars[index]
```
let's use a general name as it can be anything, but we're only modifying it if it's a packages job and the env var is `ARTIFACTS_BUCKET`

Reply:
Also added a comment on the condition

Reply:
same changes in the lines referencing this variable.